docs/content-guidelines.md

```diff
@@ -0,0 +1,65 @@
+# Kind Content guidelines
+
+We'd like to clarify that these guidelines are only effective on the
+UserStyles.world website.
+
+
+## Purpose
+
+UserStyles.world("USw") encourages their users from anyone who wishes to
+add/import/review/explore UserCSS styles("styles"), to be able to do so.
+People shouldn't feel discouraged from participating in this community.
+
+In order to ensure the safety and quality of styles, we encourage anyone
+to make a conscious effort into following these guidelines
+to the best of their ability.
+
+We'd like to note and clarify that measurements[¹](#footnote) can be taken
```

Comment:
measurements should be measures

Reply:
"measures" seems to be correct as well in this form. However I'm not sure if someone with basic English knowledge understand it the way as intended, as we don't like to use confusing wording. I myself had to check up some dictionary's to see the correct form of this. @vednoc

Reply:
Perhaps "actions" would be better, in which case "can" should be "will". We'll be taking actions if one or more guidelines are broken.

Reply:
agreed, that sounds better and is more clear as well

Reply:
All good now. The footnote explains that ^ when actions will be taken.